scripts/accumulator-server.py

```diff
@@ -40,13 +39,11 @@
 #   in the past)
 # * Curl users: use -H "Content-Type: application/xml"  for XML payload (the default:
 #   "Content-Type: application/x-www-form-urlencoded" has been problematic in the pass)
-# * This script requires at least Flask 1.0.2, which comes with Werkzeug 0.15.2. There is a bug
-#   in Werkzeug < 0.11.16 that makes empty "content-length" headers to appear for some request
-#   in the accumulator dump
-# * This script also depends on pyOpenSSL 19.0.0
-
+# * This script requires at least Flask 2.0.2, which comes with Werkzeug 2.0.2.
+#
+# TODO: test a possible bug in Werkzeug that makes empty "content-length"
+# headers to appear for some request in the accumulator dump
```

Comment:
Werkzeug 2.0.2 > 0.11.16, so I think the bug no longer exists and this two lines could be removed